policyengine_us/parameters/gov/states/nm/tax/rebates/low_income_rebate/exemption_amount1.yaml

```diff
@@ -0,0 +1,96 @@
+description: New Mexico low income comprehensive tax rebate when tax unit qualify 1 exemption
+brackets:
+  - threshold:
+      2022-01-01: 0
+    amount:
+      2022-01-01: 195
+  - threshold: 
+      2022-01-01: 1_001
+    amount:
+      2022-01-01: 220
+  - threshold:
+      2022-01-01: 7_501
+    amount:
+      2022-01-01: 205
+  - threshold:
+      2022-01-01: 8_001
+    amount:
+      2022-01-01: 185
+  - threshold: 
+      2022-01-01: 9_001
+    amount: 
+      2022-01-01: 170
+  - threshold: 
+      2022-01-01: 10_001
+    amount: 
+      2022-01-01: 145
+  - threshold: 
+      2022-01-01: 11_501
+    amount: 
+      2022-01-01: 130
+  - threshold: 
+      2022-01-01: 13_001
+    amount: 
+      2022-01-01: 115
+  - threshold: 
+      2022-01-01: 14_501
+    amount: 
+      2022-01-01: 105
+  - threshold: 
+      2022-01-01: 16_501
+    amount: 
+      2022-01-01: 100
+  - threshold: 
+      2022-01-01: 18_001
+    amount: 
+      2022-01-01: 90
+  - threshold: 
+      2022-01-01: 19_501
+    amount: 
+      2022-01-01: 80
+  - threshold: 
+      2022-01-01: 23_001
+    amount: 
+      2022-01-01: 75
+  - threshold: 
+      2022-01-01: 24_501
+    amount: 
+      2022-01-01: 65
+  - threshold: 
+      2022-01-01: 26_001
+    amount: 
+      2022-01-01: 55
+  - threshold: 
+      2022-01-01: 27_501
+    amount: 
+      2022-01-01: 50
+  - threshold: 
+      2022-01-01: 29_501
+    amount: 
+      2022-01-01: 40
+  - threshold: 
+      2022-01-01: 31_001
+    amount: 
+      2022-01-01: 35
+  - threshold: 
+      2022-01-01: 32_501
```

Comment:
we need to add the 2021 values and [source](https://klvg4oyd4j.execute-api.us-west-2.amazonaws.com/prod/PublicFiles/34821a9573ca43e7b06dfad20f5183fd/608063af-0c4c-4f28-ba42-cf40e04557ca/PITbook2021.pdf)